packages/core-data/src/hooks/use-entity-record.ts

```diff
@@ -14,11 +16,25 @@ export interface EntityRecordResolution< RecordType > {
 	/** The requested entity record */
 	record: RecordType | null;
 
+	/** The edited entity record */
+	editedRecord: Partial< RecordType >;
+
+	/** Apply edits to the edited entity record */
```

Comment:
Similar to how `save` is documented, it might be helpful to explicitly mention that those edits are applied locally.

Reply:
```suggestion
	/** Apply local (in-browser) edits to the edited entity record */
```